Automattic/jetpack
# VideoPress: disable privacy button when updating video privacy

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress:
  * Reduxify video privacy data: selector, reduce the data, expose the data from useVideo() hook, etc
  * Disable privacy button when the video privacy is updating

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to VideoPress dashboard
* Ensure you have a VideoPerss video
* Change the video privacy
* Confirm the button is disabled when updating the video privacy

https://user-images.githubusercontent.com/77539/194386864-d1d3cca1-dda4-417d-a7a2-dadc1b1ac645.mov
